src/tokenizer/stop_word_filter/mod.rs

```diff
@@ -28,95 +27,54 @@ pub struct StopWordFilter {
 }
 
 impl StopWordFilter {
+    /// Creates a new [`StopWordFilter`] for the given [`Language`]
+    ///
+    /// Returns `Some` if a list of stop words is available and `None` otherwise.
+    pub fn new(language: Language) -> Option<Self> {
```

Comment:
Can we put the feature gate on the new function directly?

Reply:
Done.